vnext/Chakra/ChakraJsiRuntime.cpp

```diff
@@ -0,0 +1,862 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "pch.h"
+
+#include "ChakraJsiRuntime.h"
+#include "ChakraJsiRuntimeArgs.h"
+
+#include <MemoryTracker.h>
+#include <cxxreact/MessageQueueThread.h>
+
+#include <mutex>
+#include <sstream>
+#include "UnicodeConversion.h"
+
+#ifdef HAS_SCRIPTSTORE
+#include <jsi/ScriptStore.h>
+#endif
+
+using namespace facebook::react;
+
+namespace facebook { 
+namespace jsi {
+namespace chakraruntime {
+
+/*static*/ std::once_flag ChakraJsiRuntime::s_runtime_version_init_flag_;

```

Comment:
>s_runtime_version_init_flag_ [](start = 44, length = 28)

s_runtime_version_init_flag?

Reply:
Unified many names to MS/Office conventions ..